src/interface/src/app/password-reset/password-reset.component.html

```diff
@@ -49,6 +59,19 @@ <h1 class="title">Reset Password</h1>
     </form>
   </div>
 
+  <mat-menu class="password-reqs-menu" #popoverMenu="matMenu">
```

Comment:
Is there a preferred way to DRY this up?

Reply:
hm good question, thing is you need the reference for matMenuTriggerFor, so not sure how you can move it to a component.

Reply:
I mean you can move the contents to a component and keep something like
```
<mat-menu class="password-reqs-menu" #popoverMenu="matMenu">
  <app-pass-reqs/>
</mat-menu>
```

Reply:
Right, I considered it, but it felt like overkill to create a new component. On the other hand, I don't love having this sort of critical, user-facing text across multiple files...I'm not sure how to handle this either, unless we wanted to implement a central strings collection, like we might have for i18n.